CONTRIBUTING.md

```diff
@@ -44,4 +44,29 @@ Fixes: https://github.com/nodejs/citgm/issues/313
 ```
 
 ## Submitting a module to CitGM
-[Read the module requirements](README.md#requirements-for-inclusion-in-nodejs-citgm-runs)
+
```

Comment:
I think some context is needed here. This is for adding a module to be included in the default citgm-all runs and not, for example, for adding dependencies to CitGM.

Also depending on the module it does not need to be added to lookup.json for CitGM to be able to test it.